phpunit/blocks/render-block-media-text-test.php

```diff
@@ -0,0 +1,128 @@
+<?php
+/**
+ * Media & Text block rendering tests.
+ *
+ * @package WordPress
+ * @subpackage Blocks
+ */
+
+/**
+ * Tests for the Media & Text block.
+ *
+ * @group blocks
+ *
+ * @covers ::render_block_core_media_text
+ */
+//class Tests_Blocks_Render_MediaText extends WP_UnitTestCase {
```

Comment:
Tiny nit - can remove

Reply:
Oh yeah I was trying to figure out what the best practice was for the naming 😺